exotica/src/Scene.cpp

```diff
@@ -729,9 +729,14 @@ namespace exotica
     return collision_scene_;
   }
 
-  std::string Scene::getRootName()
+  std::string Scene::getRootFrameName()
```

Comment:
@VladimirIvan do we even need the root of the scene exposed given that we can just do ``scene.getSolver().getRootFrameName()``?

Reply:
Not necessarily. I do like to expose scene related properties through the scene, even when they originate in the underlying solver.

Also, this PR may affect #114. I've added ```getRobotRootName``` which returns the robot root link name, instead of the scene root. It may need renaming. Feel free to modify both and merge.

Reply:
Okay, thanks. I will rename it to ``getModelRootLinkName`` if that's okay